code/drasil-example/dblpend/lib/Drasil/DblPend/DataDefs.hs

```diff
@@ -73,7 +73,7 @@ positionXQD_2 :: SimpleQDef
 positionXQD_2 = mkQuantDef xPos_2 positionXEqn_2
 
 positionXEqn_2 :: PExpr
-positionXEqn_2 = sy (positionXDD_1 ^. defLhs) `addRe` (sy lenRod_2 `mulRe` sin (sy pendDisAngle_2))
+positionXEqn_2 = sy (positionXDD_1 ^. defLhs) $+ (sy lenRod_2 $* sin (sy pendDisAngle_2))
```

Comment:
Do you think we can add precedence to these operators, so that we can avoid the extra brackets, @NoahCardoso ?

Reply:
When I add precedence it breaks other parts of the code

Reply:
Hmm, that's interesting. Can you explain a bit more about the error please?